server/routes/redirects.py

```diff
@@ -52,3 +52,9 @@ def documentation():
 @bp.route('/colab')
 def colab():
     return redirect('https://docs.datacommons.org/tutorials.html', code=302)
+
+
+@bp.route('/explore/place')
```

Comment:
please add a comment here, a warning perhaps, to not remove this (i had removed it earlier because i found no references to it from the repo).

Reply:
Maybe add a prober test for this?

Reply:
yeah, that would be great to check that the redirects continue working as expected.